src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -649,6 +649,35 @@ public void cannotCreateRequestWithoutValidUser()
       hasMessage("A valid user and patron group are required. User is null"))));
   }
 
+  @Test
+  public void cannotCreateRequestWithAnInactiveUser()
```

Comment:
Can you add a test for replacing a request with an inactive user in the `RequestsAPIUpdatingTests.java`? I am concerned that what is implemented won't work for replacing requests due to the fact that the `replace` uses `createRequest` after it does stuff to the existing request: https://github.com/folio-org/mod-circulation/blob/2e829a0729a5b4a020f481fb3c0b375782865371/src/main/java/org/folio/circulation/resources/RequestCollectionResource.java#L152-L158

Reply:
I think this scenario has now been addressed without causing issues anywhere else. Please check closely.

Reply:
@jsavell When you say it has been addressed, do you mean that the same code should be executed in both code flows and so should work?

If so, I think what @mreno-EBSCO is suggesting is to make that behaviour explicit by adding a test to demonstrate it for replacing a request.

I have just thought of a question around this. Cancelling a request is currently done by replacing it's representation. If a user is de-actived after a request is created, how does that request get addressed? Should it be possible to cancel a request for an inactive user?